Containerfile-aap

```diff
@@ -22,7 +22,9 @@ RUN set -e && pdm run python generate_embeddings-aap.py \
   -mn ${EMBEDDING_MODEL} \
   -o vector_db/aap_product_docs/${AAP_VERSION} \
   -i aap-product-docs-$(echo $AAP_VERSION | sed 's/\./_/g') \
-  -v ${AAP_VERSION}
+  -v ${AAP_VERSION} \
+  -c 200 \
```

Comment:
@justjais @jameswnl yes, this is the max chunk length parameter (`--chunks` or `-c`)

Let me try to explain how I get into that values.. has been am empiric procedure trying to fix this [VA feedback issue](https://github.com/ansible/ansible-lightspeed-va-feedback/issues/3):

1. First noticed the default value, if not specified via parameters, is `300`, that's our actual "reference"
2. I was thinking that if the bot was not able to find the right version for the ansible-rulebook, could be due to 2 miain reason: or we are including too many top relevant documents in the response (`K`),  so there the context is too much for the LLM to figure this out, or the chunks we are considering are too "big" and this small version text is not properly ingested by the LLM
3. This way I started fist by decreasing the number of top most relevant document, but noticed the default value is `K=5`, which is not much, anyway decreased to `K=3,` and then the bot was able to resolve the proper version, but on the other hand, for other more general Qs, it was not able to answer them, as it was missing context
4. This way, started to play instead with the `max_chunk_length` value, by decreasing it, trying to make the RAG to focus more on the detail of the docs,like concrete versions text, Also this will end up of smaller document chunks, but larger number of documents, so we can focus now on more specific areas on a document, rather than larger chunks. Notice the default value was `300`.
5. Initially I started with a chunk size of `150`, but then, for higher level Qs, there was missing, IMO, a few information, so to fix it, I tested then by increasing the `K` (top most relevant docs)
6. Then I did the same work with different chunk sizes, from `150` to `200`, using increases of `25` per each test

I repeated these steps manually, by then testing on each iteration the prompts and chunk sizes indicated in this PR description, until I found that the chunk_size of 200 was a good value for us, where it allows us to focus more on the details in the document, and we don't loose many context information for higher level Qs.

Also notice I changed the cuf-off value for similarity scoring in [this PR as well](https://github.com/ansible/ansible-chatbot-service/pull/41), this way we are now also just consuming context documents that are much more relevant than before. It was also an empiric process, I debugged the scoring values for different queries, and noticed that the default `0.3` was too low, so adjusted it given the empiric manual tests I did.